src/opera/test/pge/cslc_s1/test_cslc_s1_pge.py

```diff
@@ -77,6 +77,16 @@ def setUp(self) -> None:
             f"touch {join(input_dir, 'db.sqlite3')}"
         )
 
+        os.system(
```

Comment:
It would be nice to have a comment before these files that explains that they're intended for use with the `test_qa_enabled` function